### PR TITLE
feat(providers): add Trovo

### PR DESCRIPTION
**Pull request by IAmNotTheLeo**
## Checklist 🧢

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thank you, a small suggestion.

Regarding my other question, looks like we need to write an e-mail to get an app registered:

https://developer.trovo.live/docs/APIs.html#_2-register-your-application

This might take a few days once we wrote the e-mail, but I would like to wait for merging until it's done.

Unless you could provide us with a test application we could use for testing purposes? (basically to make sure that this integration will work in the future as well)

**Comment by IAmNotTheLeo** [comment]
> Thank you, a small suggestion.
> 
> Regarding my other question, looks like we need to write an e-mail to get an app registered:
> 
> https://developer.trovo.live/docs/APIs.html#_2-register-your-application
> 
> This might take a few days once we wrote the e-mail, but I would like to wait for merging until it's done.
> 
> Unless you could provide us with a test application we could use for testing purposes? (basically to make sure that this integration will work in the future as well)

I don't mind to wait until it's merged, don't have a  testing application I’m afraid :(, just want other developers to have the authentication for the streaming Platform Trovo

**Comment by balazsorban44** [comment]
This should be reopened against the `beta` branch.

@ndom91, @lluia this might need extra attention, see https://github.com/nextauthjs/next-auth/pull/2549#pullrequestreview-732011703

**Pull request closed without merging**
